.gitignore

```diff
@@ -57,6 +57,7 @@ mpi.c
 
 # ignore file generated by make tune
 tuning_list
+etc/tune
```

Comment:
@sjaeckel I hope you agree. I find this simpler and it avoids confusion. I generally use single .gitignores and only multiple if there are multiple separate projects in a repository, like a monorepo with multiple vendored projects etc.

Reply:
shouldn't this then also be etc/tuning_list ?

Reply:
Yes, but I executed tune in . and then I also wanted tuning_list to be hidden. tuning_list seems to be a unique enough name to be ignored in the whole project I thought.